test/units/Element-firstElementChild.ts

```diff
@@ -0,0 +1,16 @@
+import { DOMParser, Element } from "../../deno-dom-wasm.ts";
+import {
+  assertEquals,
+} from "https://deno.land/std@0.85.0/testing/asserts.ts";
+
+Deno.test("Node.firstElementChild", () => {
```

Comment:
Can you change the test name to "Element.firstElementChild"? Thanks

Reply:
yup, fixed